src/renderer/features/fellowship-tasks/components/Tasks.tsx

```diff
@@ -35,21 +35,19 @@ export const Tasks = () => {
   return (
     <div className="flex h-full flex-col overflow-hidden rounded-xl border border-filter-border">
       <Title count={tasksCount} />
-      {hasAccount && activeTasks.length ? (
+      {hasAccount && activeTasks.length && (
```

Comment:
At some point in time we switched to `X ? Y : null` from `X && Y`
@johnthecat can give more context